bin/utils/subkey/src/main.rs

```diff
@@ -37,6 +38,24 @@ use std::{
 mod rpc;
 mod vanity;
 
+enum OutputType {
+	Json,
+	Text
+}
+
+impl<'a> TryFrom<&'a str> for OutputType {
+	type Error = ();
+
+	fn try_from(s: &'a str) -> Result<OutputType, ()> {
```

Comment:
```suggestion
	fn try_from(s: &'a str) -> Result<OutputType, &'static str> {
```

Reply:
But this can probably left as it is, since `()` is mapped below anyway

Reply:
Cool - will leave this as is then.